### PR TITLE
Added clause on governance to ESD.3 - Simplified

### PR DESCRIPTION
**Pull request by ntnsndr**
A simpler version of my earlier attempt: https://github.com/EthicalSource/ethicalsource.dev/pull/53

### CODE REVIEW

File: content/definition.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Software is considered **Ethical Open Source** (EOS) when it meets *all* of thes
 
 1. **It benefits the commons.** The license under which the software is released must not prohibit modification, derivative works, and linking or compiling with other software (unless under a copyleft provision), in the general spirit of open source distribution.
 2. **It is created in the open.** The source code must be publicly available, developed and maintained in public view, and welcoming of public contributions (subject to review and approval by the software project’s maintainers.)
-3. **Its community is welcoming and just.** The software project’s community of maintainers and contributors must be governed by a comprehensive code of conduct that is consistently and fairly enforced.
+3. **Its community is welcoming and just.** The software project’s community of maintainers and contributors must publish clear rules for project governance, along with a comprehensive code of conduct that is consistently and fairly enforced.
```
Comment (CoralineAda) [inline content/definition.md]: Sub "and adopt" for "along with"  and I would be good with this!

**Comment by CoralineAda** [comment]
@ntnsndr bump

**Review by ntnsndr — COMMENTED** [review]
following @CoralineAda's suggestion

**Comment by ntnsndr** [comment]
Thanks for the bump! I didn't see your comment earlier.

**Review by CoralineAda — APPROVED** [review]
Love the addition.

**Pull request merged**